components/alerts/alerts.css

```diff
@@ -0,0 +1,34 @@
+.alert-red {
+  padding: 20px;
+  background-color: #f44336;
+  color: white;
+  margin-bottom: 15px;
+}
+
+.alert-darkred {
+  padding: 20px;
+  background-color: darkred;
+  color: wheat;
+  margin-bottom: 15px;
+}
+/* #ffd0d0 */
+.alert-light {
+  padding: 20px;
+  background-color: lightcoral;
+  color: darkred;
+  margin-bottom: 15px;
+}
+/* The close button */
+.closebtn {
```

Comment:
use hyphen (-) between name keep btn first and then specific type of buttton. E.g. btn-close, btn-alert, btn-primary